stock_picking_send_by_mail/tests/test_stock_picking_send_by_mail.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2017 Tecnativa <vicent.cubells@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestStockPickingSendByMail(common.TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        # Remove this variable in v16 and put instead:
+        # from odoo.addons.base.tests.common import DISABLED_MAIL_CONTEXT
+        DISABLED_MAIL_CONTEXT = {
```

Comment:
@fredericgrall Could you use this test class instead: https://github.com/odoo/odoo/blob/18.0/odoo/addons/base/tests/common.py#L21

This will avoid those lines

Reply:
The change is made